Categories/Category/Construction/Graphs.agda

```diff
@@ -199,6 +200,12 @@ Graphs o ℓ e = record
         open GraphMorphism≈
         module EqR′ {G : Graph o ℓ e} {A B : Graph.Obj G} = EqR (Graph.setoid G {A} {B})
 
+-- FIXME: for easy migration to new functor equality.  Should probably
```

Comment:
Yes, please do so if you could?  I agree that this is better, so may as well remove the old one entirely.

Reply:
Done.